src/api.v2/model/experiment.js

```diff
@@ -0,0 +1,183 @@
+const _ = require('lodash');
```

Comment:
Something weird is happening here - i dont have this file `experiment.js` locally, I have `Experiment.js`, which I modify for this PR. The even weirder thing is that it always shows that I have unstaged changes for `Experiment.js` even when I commit it and the changes i do appear in this file

Reply:
Something weird happened with the file rename

Reply:
The problem was that Mac doesnt distinguish lower and uppercase letters